python/lsst/pipe/tasks/repair.py

```diff
@@ -234,22 +234,40 @@ def cosmicRay(self, exposure, keepCRs=None):
         except Exception:
             pass
 
-        mi = exposure.getMaskedImage()
-        bg = afwMath.makeStatistics(mi, afwMath.MEDIAN).getValue()
+        exposure0 = exposure            # initial value of exposure
+        binSize = self.config.cosmicray.background.binSize
+        nx, ny = exposure.getWidth()/binSize, exposure.getHeight()/binSize
+        if nx*ny <= 1:
```

Comment:
I see the logic here, but: what happens if we call ``estimateBackground`` even when ``nx*ny`` is less than 1?

* If it does just use the median, can we just call it here? It simplifies the logic since we don't need to handle this case specially.
* If it doesn't just use the median, what's the justification for using the same technique here as in ``estimateBackground`` for small bins, and a different technique for large bins?